src/map/clif.cpp

```diff
@@ -10803,6 +10833,8 @@ void clif_parse_LoadEndAck(int fd,map_session_data *sd)
 		clif_send_homdata( *sd->hd, SP_ACK );
 		clif_hominfo(sd,sd->hd,1);
 		clif_hominfo(sd,sd->hd,0); //for some reason, at least older clients want this sent twice
+		clif_homunculus_updatestatus(*sd, SP_HP);
+		clif_homunculus_updatestatus(*sd, SP_SP);
```

Comment:
Why do we need to resend here?
HP/SP are part of clif_hominfo.

Reply:
Official server does 🤷
In fact, they send those most of the times after sending hominfo twice